switchcraft/data/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -92,8 +94,12 @@ local function get(url)
 
   -- Add a cache buster so that Pastebin spam protection is re-checked
   local cacheBuster = ("%x"):format(math.random(0, 2 ^ 30))
-  local response, err, errResponse = http.get(rawUrl .. "?cb=" .. cacheBuster)
-
+  local response, err, errResponse
+  if pw then
+    response, err, errResponse = http.get(rawUrl .. "?cb=" .. cacheBuster.."&password="..pw)
```

Comment:
Sorry, I missed a comment in my previous review. Can you confirm whether this works for Pastebin, or only SCPaste? My understanding is that password-protected pastes on Pastebin require a form submission with a CSRF token, and a different parameter. If they do not work, the program should respond clearly with an error message indicating that passwords are not supported for Pastebin.

Reply:
Have not tested with actual pastebin, will do